tests/e2e/framework/kubernetes.go

```diff
@@ -90,36 +80,41 @@ func newKubeInfo(tmpDir, runID string) *KubeInfo {
 
 	return &KubeInfo{
 		Namespace:        *namespace,
-		NamespaceCreated: false,
-		AppHub:           *appHub,
-		AppTag:           *appTag,
-		MixerHub:         *mixerHub,
-		MixerTag:         *mixerTag,
-		ManagerHub:       *managerHub,
-		ManagerTag:       *managerTag,
-		CaHub:            *caHub,
-		CaTag:            *caTag,
-		ProxyHub:         *managerHub,
-		ProxyTag:         *managerTag,
+		namespaceCreated: false,
+		MixerImage:       fmt.Sprintf("%s/mixer:%s", *mixerHub, *mixerTag),
+		ManagerImage:     fmt.Sprintf("%s/manager:%s", *managerHub, *managerTag),
+		CaImage:          fmt.Sprintf("%s/ca:%s", *caHub, *caTag),
+		ProxyImage:       fmt.Sprintf("%s/proxy:%s", *managerHub, *managerTag),
```

Comment:
Maybe add a comment here explain why we are giving manager hub/tag to proxy, i.e. manager and proxy always share the same.